src/TestProblem/Hydro/CCSN/Init_TestProb_Hydro_CCSN.cpp

```diff
@@ -70,6 +70,13 @@ double Mis_GetTimeStep_Lightbulb( const int lv, const double dTime_dt );
 double Mis_GetTimeStep_CoreCollapse( const int lv, const double dTime_dt );
 bool   Flag_CoreCollapse( const int i, const int j, const int k, const int lv, const int PID, const double *Threshold );
 bool   Flag_Lightbulb( const int i, const int j, const int k, const int lv, const int PID, const double *Threshold );
+bool   Flu_ResetByUser_Func_CCSN( real fluid[], const double x, const double y, const double z, const double Time,
+                                 const double dt, const int lv, double AuxArray[] );
+void   Flu_ResetByUser_API_CCSN( const int lv, const int FluSg, const double TimeNew, const double dt );
+
+// this test problem needs to reset both Flu_ResetByUser_API_Ptr and Flu_ResetByUser_Func_Ptr, while
```

Comment:
The `Migration Test` need not to reset these two pointers.

Reply:
I revised the comment a bit.